experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -594,13 +596,36 @@ function FluxCalculator.compute_surface_fluxes!(
     @. csf.F_turb_moisture += csf.temp1 * area_fraction
 
     # Combine turbulent momentum fluxes from each component of the land model
+    # Note that we exclude the canopy component here for now, since we can have nonzero momentum fluxes
+    #  where there is zero LAI. This should be fixed in ClimaLand.
     @. csf.temp1 = soil_dest.ρτxz * (1 - p.snow.snow_cover_fraction) + p.snow.snow_cover_fraction * snow_dest.ρτxz
     @. csf.temp1 = ifelse(area_fraction == 0, zero(csf.temp1), csf.temp1)
     @. csf.F_turb_ρτxz += csf.temp1 * area_fraction
 
     @. csf.temp1 = soil_dest.ρτyz * (1 - p.snow.snow_cover_fraction) + p.snow.snow_cover_fraction * snow_dest.ρτyz
     @. csf.temp1 = ifelse(area_fraction == 0, zero(csf.temp1), csf.temp1)
     @. csf.F_turb_ρτyz += csf.temp1 * area_fraction
+
+    # Combine the buoyancy flux from each component of the land model
+    # Note that we exclude the canopy component here for now, since ClimaLand doesn't
+    #  include its extra resistance term in the buoyancy flux calculation.
+    @. csf.temp1 =
+        soil_dest.buoy_flux * (1 - p.snow.snow_cover_fraction) + p.snow.snow_cover_fraction * snow_dest.buoy_flux
+    @. csf.temp1 = ifelse(area_fraction == 0, zero(csf.temp1), csf.temp1)
+    @. csf.buoyancy_flux += csf.temp1 * area_fraction
+
+    # Compute ustar from the momentum fluxes and surface air density
```

Comment:
Maybe add a formula?

Reply:
I added the formulas we're using for ustar and L_MO